spec/features/reviews/user_votes_on_review_spec.rb

```diff
@@ -22,37 +22,52 @@
       click_button 'Log in'
     end
 
-    scenario "visit details page, clicks upvote button" do
+    scenario "visit details page, clicks upvote button", js: true do
       review = FactoryGirl.create(:review)
 
       visit dinosaur_path(review.dinosaur)
-      click_link("Upvote")
+      page.find('.upvote-button').click
 
-      votes = review.get_upvotes.size
-      expect(votes).to eq(1)
+      expect(page).to have_content('1')
     end
 
-    scenario "visit details page, clicks downvote button" do
+    scenario "visit details page, clicks downvote button", js: true do
       review = FactoryGirl.create(:review)
 
       visit dinosaur_path(review.dinosaur)
-      click_link("Downvote")
-      votes = review.get_downvotes.size
-      expect(votes).to eq(1)
+      page.find('.downvote-button').click
+
+      expect(page).to have_content('-1')
     end
 
-    scenario "visit details page, clicks Unvote button" do
+    scenario "visit details page, clicks Unvote button", js: true do
       review = FactoryGirl.create(:review)
 
       visit dinosaur_path(review.dinosaur)
-      click_link("Upvote")
-      votes = review.get_upvotes.size
-      expect(votes).to eq(1)
+      page.find('.downvote-button').click
+      page.find('.unvote-button').click
 
-      click_link("Unvote")
-      votes = review.get_upvotes.size
-      expect(votes).to eq(0)
+      expect(page).to have_content('0')
     end
 
+    scenario "user can't upvote more than once", js: true do
+      review = FactoryGirl.create(:review)
+
+      visit dinosaur_path(review.dinosaur)
+      page.find('.upvote-button').click
+      page.find('.upvote-button').click
+
+      expect(page).to have_content('1')
+    end
+    
```

Comment:
Yeah, Hound isn't barking about the blank line, it's actually two spaces on line 62.